lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -67,7 +68,7 @@ def add_logger(app)
           # Instead, we patch Lograge `custom_options` internals directly
           # as part of Rails framework patching
           # and just flag off the warning log here.
-          should_warn = false if app.config.respond_to?(:lograge)
+          should_warn = false if app.config.respond_to?(:lograge) || defined?(::SemanticLogger)
```

Comment:
I'm not too familiar with this line, so I read the comment above and it refers to lograre and `custom_options`.

How does this apply to sematic logger?

Or I guess a better request: do you think it makes sense to update the comment above to explain why this is needed for semantic logger as well?